### PR TITLE
Use correct port mappings for external listeners

### PR DESCRIPTION
**Pull request by OneCricketeer**
**Description of the change**

Update docs 

**Benefits**

It'll work as expected, now

### CODE REVIEW

**Comment by FraPazGal** [comment]
Hi @OneCricketeer,

Thank you for the PR! At first glance, I don't understand why are you removing the port `9092` from the `docker-compose.yaml` and interchanging the ports in `kafka-console-producer.sh --broker-list 127.0.0.1:9092 --topic test` between internal and external client. 

Could you please explain why the current workflow description is wrong or reference a related issue?

**Comment by OneCricketeer** [comment]
If you connect to 9092, then it'll return `kafka:9092` from the internal advertised listener and fail to connect from the external host

**Comment by OneCricketeer** [comment]
The section headers need switched as well

File: README.md [hunk 1]
```diff
@@ -279,17 +281,17 @@ And expose the extra port:
 These clients will use localhost to connect to Kafka.
 
 ```console
-kafka-console-producer.sh --broker-list 127.0.0.1:9092 --topic test
-kafka-console-consumer.sh --bootstrap-server 127.0.0.1:9092 --topic test --from-beginning
+kafka-console-producer.sh --broker-list 127.0.0.1:9093 --topic test
+kafka-console-consumer.sh --bootstrap-server 127.0.0.1:9093 --topic test --from-beginning
 ```
 
 #### Producer and consumer using external client
```
Comment (andresbono) [inline README.md:288]: As you mentioned in a previous comment, the headings need to be switched, right?
```suggestion
#### Producer and consumer using internal client
```

Could you change the other heading as well?

**Comment by OneCricketeer** [comment]
Updated @andresbono

**Review by andresbono — APPROVED** [review]
Perfect, thanks for addressing the suggestions!

**Pull request merged**